src/Microsoft.Framework.Runtime/Extensions/JTokenExtensions.cs

```diff
@@ -0,0 +1,25 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace Newtonsoft.Json.Linq
+{
+    internal static class JTokenExtensions
```

Comment:
Don't we have a place that has the other extension method `ValueAsArray<T>` put this there